### ISSUE TITLE
CT: Cannot read property 'addToBack' of undefined

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gravity-force-lab#272]
This appears to have started on 10/20/20 @ 12:40pm, and is failing on every cycle of CT for GLF and GFL:B.

```
gravity-force-lab : interactive-description-fuzzBoard : built
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000
Query: fuzzBoard&supportsDescriptions&memoryLimit=1000
Uncaught TypeError: Cannot read property 'addToBack' of undefined
TypeError: Cannot read property 'addToBack' of undefined
at pF.endDrag [as _endChange] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:2336189)
at pF.onInteractionEnd (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:1149165)
at pF.handleKeyUp (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:1148354)
at Lh.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:575581)
at Lh.dispatchToTargets (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:575756)
at Lh.dispatchEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:575346)
at Lh.dispatchA11yEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:571757)
at Lh.keyupAction.Mr.phetioPlayback (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:569016)
at Ir.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:379185)
at HTMLDivElement.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603240241484/gravity-force-lab/build/phet/gravity-force-lab_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab%22%2C%22interactive-description-fuzzBoard%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603240241484%22%2C%22timestamp%22%3A1603241988801%7D&fuzzBoard&supportsDescriptions&memoryLimit=1000:854:569409)
id: Bayes Chrome
Snapshot from 10/20/2020, 6:30:41 PM
```

GFL:B is sometimes failing with:

```
gravity-force-lab-basics : fuzz : built
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000
Query: fuzz&memoryLimit=1000
Uncaught TypeError: Cannot set property 'enabled' of undefined
TypeError: Cannot set property 'enabled' of undefined
at Wx.down (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:1102125)
at Ja.dispatchToListeners (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:503235)
at Ja.dispatchToTargets (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:503410)
at Ja.dispatchEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:503000)
at Ja.downEvent (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:502087)
at Ja.touchStartAction.Hn.phetioPlayback (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:491269)
at Nn.execute (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:340386)
at Ja.touchStart (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:500052)
at PA.touchStart (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:846238)
at PA.touchStartAction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1603230328741/gravity-force-lab-basics/build/phet/gravity-force-lab-basics_en_phet.html?continuousTest=%7B%22test%22%3A%5B%22gravity-force-lab-basics%22%2C%22fuzz%22%2C%22built%22%5D%2C%22snapshotName%22%3A%22snapshot-1603230328741%22%2C%22timestamp%22%3A1603231621185%7D&fuzz&memoryLimit=1000:858:844471)
id: Bayes Chrome
Snapshot from 10/20/2020, 3:45:28 PM
```

**Comment by pixelzoom** [comment]
This is also occurring in coulombs-law and john-trovoltage. So it may be appropriate to transfer this issue to a general repo.

**Comment by jessegreenberg** [comment]
This was fixed in phetsims/utterance-queue#15
